doc/content/getting-started/guides/build-an-end-to-end-solution-gn/_index.md

```diff
@@ -0,0 +1,276 @@
+- In {{% tts %}} **Console** select **Applications**.
+- Select the **Create application** button.
+- On the **Create application** page provide the following information:
+  - **Application ID:** A unique identification for your application. Eg. _yourname-climate-tracker_
+  - **Application name:** Any descriptive name. Eg. _[Your Name]'s Room Climate Tracker_
+- Select the **Create application** button.
+
+{{< figure src="create-application.png" alt="" >}}
+
+- Once created you will be forwarded to the **overview** page of the application you just created.
+
+# Onboarding Generic Node SE
+
+Generic Node Sensor Edition is a hardware platform that simplifies building LoRaWAN® end-devices for various use cases. It provides a fast development lifecycle from concept to production with its reference hardware design files and source-available firmware.
+
+{{< figure src="generic-node-se.png" alt="" >}}
+
+An end device can be added to {{% tts %}} through [manual onboarding](https://www.thethingsindustries.com/docs/devices/adding-devices/#manually-registering-an-end-device), [device repository](https://www.thethingsindustries.com/docs/devices/adding-devices/#using-the-lorawan-device-repository), or [scanning a QR code](https://www.thethingsindustries.com/docs/devices/adding-devices/#onboarding-devices-using-qr-codes).
+
+In this project, we will add the Generic Node to {{% tts %}} by scanning its QR code. The QR code for claiming the Generic Node is located on the back of its enclosure. For complete steps on how to claim a device using a QR code, please refer to the instructions available [here](https://www.thethingsindustries.com/docs/devices/adding-devices/#onboarding-devices-using-qr-codes).
+
+{{< figure src="generic-node-qr-code.png" alt="" >}}
+
+Once claimed you will be forwarded to the **View registered end device** page.
+
+{{< note "Your Generic Node Sensor Edition comes pre-programmed with a [vanilla](https://www.genericnode.com/docs/applications/se-vanilla/) application that can be used with this project. The full instructions on how to program Generic Node SE can be found [here](https://www.genericnode.com/docs/getting-started/se-sw/)." />}}
+ 
+- Go to the **Live data** tab.
+- Power cycle your Generic Node by either pressing the reset button or removing and re-inserting its batteries.
+- Once power cycled, you can immediately see the activity status such as **Accept join-request** and **Forward join-accept messages** in the **Live data** section.
+- Now press the user button on top of the Generic Node.
+- In the **Live data** section, you can see an uplink data message received by {{% tts %}} application. 
+- Select the uplink message. You can see the detailed view of your message in the **Event details** window. Scroll down and see what’s in the **frm_payload** (frame payload) field. The payload is encoded which is not useful until it is decoded.
```

Comment:
Payload is maybe not comprehensible to every human, but you can't say it's not useful. Please reword this and link payload formatting section